src/chapters/booleans-and-conditionals/logical-operators.rst

```diff
@@ -0,0 +1,273 @@
+
+JavaScript allows us to create a compound boolean expression using the logical AND operator, ``&&``. A compound boolean expression is a boolean expression built out of smaller boolean expressions. 
+
+The operator takes two operands, and the resulting expression is ``true`` if *both* operands are ``true`` individually. If either operand is ``false``, the overall expression is ``false``.
+
+In English, the ``&&`` operator mirrors the use of the word "and" (hence the name "logical AND"). A sentence like "Roses are red and violets are blue," is true as a whole precisely because roses are actually red, and violets are actually blue. On the other hand, the sentence "Roses are red and violets are green," is false as a whole. While roses are indeed red, violets are not green. 
+
+Let's see how this works in code.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+      :linenos:
+
+      console.log(7 > 5 && 5 > 3);
+      console.log(7 > 5 && 2 > 3);
+      console.log(2 > 3 && 'dog' === 'cat');
+
+   **Output**
+
+   ::
+
+      true
+      false
+      false
+
+In line 1, ``7 > 5 && 5 > 3`` returns ``true`` because both ``7 > 5`` and ``5 > 3`` are ``true`` individually.
+
+In line 2, ``7 > 5 && 2 > 3`` returns ``false`` because one of the two expressions, ``2 > 3``, is ``false``.
+
+Like line 2, line 3 returns ``false`` because both sub-expressions are ``false``. Notice in this example that we can mix and match data types however we like, as long as both sides of the ``&&`` expression are themselves boolean expressions.
+
+Logical OR
+^^^^^^^^^^
+
+.. index:: ! ||
+
+.. index::
+   single: operator; boolean
+
+JavaScript's logical OR operator, ``||``, also creates compound boolean expressions. This operator takes two operands, and the resulting expression is ``true`` if *either* of the operands are ``true`` individually. If both operands are ``false``, the overall expression is ``false``.
+
+As with logical AND, logical OR mirrors our experience of English language truth values. The sentence "Pigs can fly or dogs can run," is true as a whole. Joining the two clauses by "or" requires that only one of them is true in order for the full sentence to be true. Similarly, "Pigs can fly or dogs can speak Spanish," is a false statement.
+
+Let's look at some examples in JavaScript.
+
+.. sourcecode:: js
+   :linenos:
+
+   console.log(7 > 5 || 5 > 3);
+   console.log(7 > 5 || 2 > 3);
+   console.log(2 > 3 || 'dog' === 'cat');
+
+**Output**
+
+::
+
+   true
+   true
+   false
+
+Lines 1 and 2 both return ``true`` because at least one of the comparison expressions joined by ``||`` is ``true``. Line 3 returns ``false`` because both sub-expressions are ``false``.
+
+.. warning:: The single symbols ``&`` and ``|`` are themselves valid JavaScript operators, so accidentally leaving off one of the symbols off when typing ``&&`` or ``||`` will not result in an error. The resulting value of the expression will be different than expected though.
```

Comment:
Phrasing: Retain only one 'off' in the "...leaving" phrase.  Consider replacing 'themselves' with 'also'.

Phrasing: "However, the result will be different than expected."

Following this warning, a concrete example of using & vs. && or | vs. || would be helpful.  Alternatively, encourage the students to try for themselves on repl.it.

Reply:
Since we aren't covering the bitwise operators, `&` and `|`, I'm not sure what an example would show. Perhaps the point could be better made with by emphasizing or rephrasing portions? The main points here are:
1. `&` and `&&` are different, as are `||` and `|`
2. Don't use `&` or `|`

Reply:
Ah, now I see:
let num = 55, num2 = 48;

console.log(num > 50 && num2 < 100);  Prints ``true`` or ``false`` as num & num2 change.
console.log(num > 50 & num2 < 100);  Prints 1 or 0 as num & num2 change.

The difference is subtle.  The question is whether ``true`` vs. 1 change the behavior of an if/else statement, a while loop or a return value.  My guess is that in most cases (especially novice coding), the results will be the same?

Perhaps the end statement could be made more specific.  How about, "The resulting value of the expression will be ``1`` or ``0`` instead of ``true`` or ``false``."  Then follow with some reasoning for why T/F is the way to go.

Alternative: "The single symbols ``&`` and ``|`` are themselves valid JavaScript operators, but using them produces values of ``1`` or ``0`` instead of ``true`` or ``false``.  If your code expects a boolean, then use ``&&`` and ``||``.  Using ``&`` or ``|`` MIGHT work, but it opens your code up for errors."